content/en/smart-contracts/fundamentals/the-filecoin-virtual-machine/index.md

```diff
@@ -23,222 +23,44 @@ aliases:
     - "/developers/smart-contracts/about/use-cases/"
 ---
 
-## Features
+## Introduction
 
-The Filecoin virtual machine (FVM) was created to enable developers to build new use cases on top of the Filecoin network. Data access control, data DAOs, perpetual storage, collateral leasing, and Ethereum-compatible fungible and non-fungible tokens are just some of the use-cases for the FVM.
+Filecoin’s storage and retrieval capabilities can be thought of as the base layer of the Filecoin blockchain, and FVM can be thought of as a layer on top of Filecoin enabling programmatic execution of storage and retrieval.  
 
-### Data access control
+Whereas other blockchains do have smart contract capabilities, FVM’s smart contracts can use Filecoin storage and retrieval primitives with computational logic conditions.  FVM will also enable Layer 2 capabilities, such as “compute over data” and [content delivery networks](https://saturn.tech/).
```

Comment:
Simplify this paragraph. And make it less buzz-wordy. _Computational logic conditions_ is far to technical.

> FVM will also enable Layer 2 capabilities,

Talk about what the FVM can do _now_, not later -- that's what the roadmap is for.

Reply:
Ok, you can change the tense here, no problem.